core/src/main/java/com/datastrato/gravitino/catalog/CatalogManager.java

```diff
@@ -162,7 +162,7 @@ public void close() {
         classLoader.withClassLoader(
             cl -> {
               if (catalog != null) {
-                catalog.ops().close();
+                catalog.close();
```

Comment:
Why don't you add a getter method in `BaseCatalog` and close it if the return value is not null? do you have any other thoughts?

Reply:
the method `ops()` in `BaseCatalog` already means the getter method, it's weird to add a getter method again